### DISCUSSION TITLE
Hide navigation sidebar contents but retain the whitespace?

### DISCUSSION THREAD

**Discussion by Stephen-Gates** [squidfunk/mkdocs-material#3547] [Q&A] [answered]
Along similar lines as https://github.com/squidfunk/mkdocs-material/discussions/3405, is it possible to hide the contents of the navigation  but retain the whitespace of the sidebar? 

Most of my pages show the navigation and table of contents

<img width="1440" alt="Screen Shot 2022-02-07 at 6 52 03 am" src="https://user-images.githubusercontent.com/9379524/152700870-6748f9bb-8fb3-4a32-a802-46152bf4c4cb.png">

Some pages have only themselves in the navigation

<img width="1440" alt="Screen Shot 2022-02-07 at 6 28 54 am" src="https://user-images.githubusercontent.com/9379524/152700896-1d1f5ede-7af2-436b-a5ae-84f2cf6110f2.png">

Some pages have lots of entries in the navigation but the page is playing the role of the navigation

<img width="1440" alt="Screen Shot 2022-02-07 at 6 29 29 am" src="https://user-images.githubusercontent.com/9379524/152700933-27c40bb0-a844-48ca-9450-a9c54dcf5be3.png">

I know I can hide the navigation but when I do the main content shifts left, which is a little jarring when you go from page to page and the content shifts around.  

<img width="1440" alt="Screen Shot 2022-02-07 at 6 39 25 am" src="https://user-images.githubusercontent.com/9379524/152700956-14b357e7-8d2b-4c69-843c-c9ef20f831e3.png">

Is it possible to Hide the navigation sidebar contents but retain its whitespace?

**Comment by squidfunk** [comment, marked as answer]
Yes, achievable with CSS:

``` css
/*  Navigation sidebar */
.md-nav--primary {
  display: none
}

/*  Table of contents sidebar */
.md-nav--secondary {
  display: none
}
```

**Reply by Stephen-Gates** [reply]
Thanks for the quick reply and used the https://github.com/squidfunk/mkdocs-material/discussions/3220 answer to apply this just to the required pages. 🙏
